app/src/main/java/sdp/moneyrun/database/GameDatabaseProxy.java

```diff
@@ -77,6 +78,19 @@ public String putGame(@Nullable Game game) {
         return id;
     }
 
+    /**
+     * Remove a game to the database.
+     *
+     * @param game the game to be removed in the database
+     */
+    public void removeGame(@Nullable Game game) {
+        if (game == null) {
```

Comment:
This should also be tested, at least the case where game is null

Reply:
done